articles/tokens/access-token.md

```diff
@@ -107,9 +114,39 @@ For more information on the namespaced format of custom claims, refer to [User p
 
 For an example of how to add a custom claim, refer to [Add Custom Claims](/scopes/current#example-add-custom-claims).
 
-## Authorize Access Tokens
+## Lifetime
+
+The token lifetime can be controlled on a per-API basis. The validity period can be increased or decreased based on the security requirements of each API.
+
+To configure the amount of time a token lives, use the **Token Expiration (Seconds)** field for your API at the [APIs dashboard](${manage_url}/#/apis). The default value is `24` hours (`86400` seconds).
+
+![Token Expiration - API](/media/articles/tokens/tokens-expiration-api.png)
+
+Once expired, an access token can no longer be used to access an API. In order to obtain access again, a new access token needs to be obtained. This can be done by repeating the OAuth flow used to obtain the initial access token.
+
+In some situations, it is desirable to have permanent, ongoing access to an API without having to repeat an OAuth flow. This is often referred to as `offline access`, and is possible with the use of a [refresh token](/tokens/refresh-token).
+
+A refresh token is issued from the OAuth 2.0 endpoints along with the access token. When the access token expires, the refresh token can be used to obtain a fresh access token with the same permissions, without further involvement from a user. Note that offline access is enabled as a policy of the API the access token grants access to. If the API does not permit offline access, a refresh token will not be issued. In such circumstances, the OAuth flow must be repeated in order to obtain a new access token.
```

Comment:
`Note that offline access is enabled as a policy of the API the access token grants access to. If the API does not permit offline access, a refresh token will not be issued.`

You do not tell them how they can check and change this (field at API settings)